src/OmniSharp/MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -197,6 +205,12 @@ public static ProjectFileInfo Create(MSBuildOptions options, ILogger logger, str
                     .Select(p => Path.GetFullPath(Path.Combine(projectFileInfo.ProjectDirectory, p.FinalItemSpec)))
                     .ToList();
 
+                var allowUnsafe = properties["AllowUnsafeBlocks"].FinalValue;
```

Comment:
@filipw This is a problem when a csproj file doesn't contain that property. See #314